src/common/records.lua

```diff
@@ -53,7 +53,21 @@ end
 --- Get all records from the ANT
 ---@return string The encoded JSON representation of all records.
 function records.getRecords()
-	return json.encode(Records)
+	local antRecords = utils.deepCopy(Records)
+	assert(antRecords, "Failed to copy Records")
```

Comment:
what is this protecting against?

Reply:
deepCopy returns table | nil

Reply:
please add unit tests for this

Reply:
[5a080c7](https://github.com/ar-io/ar-io-ant-process/pull/35/commits/5a080c72175a6e604b290ae89e5bd53fc3cbbee1)